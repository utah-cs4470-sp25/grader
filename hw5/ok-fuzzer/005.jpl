let P__1 = u3_6 { I__0m { V_01_ ( y,h,S ) . Z9_1 },V_27 ( ( if 355044537 then E else 157520417 % 1169337049 ),sB,array [ Z__5_ : gXIw6,a5 : if 36772519 then 603780457 else B . W,ie : - B,f0_2X : v0_,T : 742399 . Ho8P9N ] 189 . I_e17d,[ X,9370918687,( T ),K,B ] . E_Oej8 ),[ array [ IX : n,P_4b : N ] ( p ),- B == J . VI_,B05_ [ T . n ],7138835359 . k__Nd . LoL13e ] [ 4 ] }
print "!<3"//p4$__N4
  
print "Byz_A9_0_"
  
write image r_g_2_ to "_pV0LY/U_7" 
//t1G1'~99|
assert o_E76 ( ( K_m29 { O,784469596,( w ) } [ ! i,z4E_A,D < 13761,H ] ),sum [ E_Q : if ( z ) then d else 107482,r676j : 2469609214,Q6o_6A : 37420,d26Z : sum [ y : z ] o ] c8_3 . u,eE ( j,D * e ) . C2VU [ lEh { c,( e ) },1,S >= H . u3a . T9___E,sum [ r0_h4 : J . h,M0__2_ : 823,k7i_ : array [  ] c,X99z : k ] n . i [ Q ],i . r [ I . R,O . r,14 ] ],E_3c { sum [ D815 : array [  ] h,O0_5 : O ] ( A ),q4i_ ( sum [  ] D,j ),if array [  ] v then - C else if ( g ) then o . X else k,2979866,101 } ) . f1t , "__,"//_4\Db _,_?1_w__8_4R

print "XI:}"   
//q\_8"__8__3674_z_*
