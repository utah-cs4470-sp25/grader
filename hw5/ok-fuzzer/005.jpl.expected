(LetCmd (VarLValue P__1) (StructLiteralExpr u3_6 (StructLiteralExpr I__0m (DotExpr (CallExpr V_01_ (VarExpr y) (VarExpr h) (VarExpr S)) Z9_1)) (CallExpr V_27 (IfExpr (IntExpr 355044537) (VarExpr E) (BinopExpr (IntExpr 157520417) % (IntExpr 1169337049))) (VarExpr sB) (ArrayLoopExpr Z__5_ (VarExpr gXIw6) a5 (IfExpr (IntExpr 36772519) (IntExpr 603780457) (DotExpr (VarExpr B) W)) ie (UnopExpr - (VarExpr B)) f0_2X (VarExpr v0_) T (DotExpr (IntExpr 742399) Ho8P9N) (DotExpr (IntExpr 189) I_e17d)) (DotExpr (ArrayLiteralExpr (VarExpr X) (IntExpr 9370918687) (VarExpr T) (VarExpr K) (VarExpr B)) E_Oej8)) (ArrayIndexExpr (ArrayLiteralExpr (ArrayLoopExpr IX (VarExpr n) P_4b (VarExpr N) (VarExpr p)) (BinopExpr (UnopExpr - (VarExpr B)) == (DotExpr (VarExpr J) VI_)) (ArrayIndexExpr (VarExpr B05_) (DotExpr (VarExpr T) n)) (DotExpr (DotExpr (IntExpr 7138835359) k__Nd) LoL13e)) (IntExpr 4))))
(PrintCmd "!<3")
(PrintCmd "Byz_A9_0_")
(WriteCmd (VarExpr r_g_2_) "_pV0LY/U_7")
(AssertCmd (DotExpr (CallExpr o_E76 (ArrayIndexExpr (StructLiteralExpr K_m29 (VarExpr O) (IntExpr 784469596) (VarExpr w)) (UnopExpr ! (VarExpr i)) (VarExpr z4E_A) (BinopExpr (VarExpr D) < (IntExpr 13761)) (VarExpr H)) (SumLoopExpr E_Q (IfExpr (VarExpr z) (VarExpr d) (IntExpr 107482)) r676j (IntExpr 2469609214) Q6o_6A (IntExpr 37420) d26Z (SumLoopExpr y (VarExpr z) (VarExpr o)) (DotExpr (VarExpr c8_3) u)) (ArrayIndexExpr (DotExpr (CallExpr eE (VarExpr j) (BinopExpr (VarExpr D) * (VarExpr e))) C2VU) (StructLiteralExpr lEh (VarExpr c) (VarExpr e)) (IntExpr 1) (BinopExpr (VarExpr S) >= (DotExpr (DotExpr (VarExpr H) u3a) T9___E)) (SumLoopExpr r0_h4 (DotExpr (VarExpr J) h) M0__2_ (IntExpr 823) k7i_ (ArrayLoopExpr (VarExpr c)) X99z (VarExpr k) (ArrayIndexExpr (DotExpr (VarExpr n) i) (VarExpr Q))) (ArrayIndexExpr (DotExpr (VarExpr i) r) (DotExpr (VarExpr I) R) (DotExpr (VarExpr O) r) (IntExpr 14))) (StructLiteralExpr E_3c (SumLoopExpr D815 (ArrayLoopExpr (VarExpr h)) O0_5 (VarExpr O) (VarExpr A)) (CallExpr q4i_ (SumLoopExpr (VarExpr D)) (VarExpr j)) (IfExpr (ArrayLoopExpr (VarExpr v)) (UnopExpr - (VarExpr C)) (IfExpr (VarExpr g) (DotExpr (VarExpr o) X) (VarExpr k))) (IntExpr 2979866) (IntExpr 101))) f1t) "__,")
(PrintCmd "XI:}")
